dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.auditWebProbes-1.0.javaee.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName =io.openliberty.auditWebProbes-1.0.javaee
```

Comment:
Seems convention is to remove the '-' in auto features. Applies to the name as well.
```suggestion
symbolicName =io.openliberty.auditWebProbes1.0.javaee
```